### ISSUE TITLE
Missing checksums cause failures

### ISSUE THREAD

**Issue by lawnmowerlatte** [deimosfr/packer-debian#4]
Checksums seem to be required based on [hashicorp/packer#2722](https://github.com/hashicorp/packer/issues/2722) I'm getting the same error reported there:

```
$ packer build debian-jessie
virtualbox-iso output will be in this color.

Warnings for build 'virtualbox-iso':

* A checksum type of 'none' was specified. Since ISO files are so big,
a checksum is highly recommended.

==> virtualbox-iso: Downloading or copying Guest additions
    virtualbox-iso: Downloading or copying: file:///Applications/VirtualBox.app/Contents/MacOS/VBoxGuestAdditions.iso
==> virtualbox-iso: Downloading or copying ISO
    virtualbox-iso: Downloading or copying: http://cdimage.debian.org/cdimage/release/8.5.0/amd64/iso-cd/debian-8.5.0-amd64-netinst.iso
==> virtualbox-iso: Starting HTTP server on port 8254
==> virtualbox-iso: Creating virtual machine...
==> virtualbox-iso: Creating hard drive...
==> virtualbox-iso: Error attaching ISO: VBoxManage error: VBoxManage: error: Could not get the storage format of the medium '/Users/lawnmowerlatte/packer-debian/packer_cache/71496103e631b73270b67cf5011b37e64a93b062e305482536cc720ffea678c6.iso' (VERR_NOT_SUPPORTED)
==> virtualbox-iso: VBoxManage: error: Details: code VBOX_E_IPRT_ERROR (0x80bb0005), component MediumWrap, interface IMedium, callee nsISupports
==> virtualbox-iso: VBoxManage: error: Context: "OpenMedium(Bstr(pszFilenameOrUuid).raw(), enmDevType, enmAccessMode, fForceNewUuidOnOpen, pMedium.asOutParam())" at line 179 of file VBoxManageDisk.cpp
==> virtualbox-iso: VBoxManage: error: Invalid UUID or filename "/Users/lawnmowerlatte/packer-debian/packer_cache/71496103e631b73270b67cf5011b37e64a93b062e305482536cc720ffea678c6.iso"
==> virtualbox-iso: Unregistering and deleting virtual machine...
==> virtualbox-iso: Deleting output directory...
Build 'virtualbox-iso' errored: Error attaching ISO: VBoxManage error: VBoxManage: error: Could not get the storage format of the medium '/Users/lawnmowerlatte/packer-debian/packer_cache/71496103e631b73270b67cf5011b37e64a93b062e305482536cc720ffea678c6.iso' (VERR_NOT_SUPPORTED)
VBoxManage: error: Details: code VBOX_E_IPRT_ERROR (0x80bb0005), component MediumWrap, interface IMedium, callee nsISupports
VBoxManage: error: Context: "OpenMedium(Bstr(pszFilenameOrUuid).raw(), enmDevType, enmAccessMode, fForceNewUuidOnOpen, pMedium.asOutParam())" at line 179 of file VBoxManageDisk.cpp
VBoxManage: error: Invalid UUID or filename "/Users/lawnmowerlatte/packer-debian/packer_cache/71496103e631b73270b67cf5011b37e64a93b062e305482536cc720ffea678c6.iso"

==> Some builds didn't complete successfully and had errors:
--> virtualbox-iso: Error attaching ISO: VBoxManage error: VBoxManage: error: Could not get the storage format of the medium '/Users/lawnmowerlatte/packer-debian/packer_cache/71496103e631b73270b67cf5011b37e64a93b062e305482536cc720ffea678c6.iso' (VERR_NOT_SUPPORTED)
VBoxManage: error: Details: code VBOX_E_IPRT_ERROR (0x80bb0005), component MediumWrap, interface IMedium, callee nsISupports
VBoxManage: error: Context: "OpenMedium(Bstr(pszFilenameOrUuid).raw(), enmDevType, enmAccessMode, fForceNewUuidOnOpen, pMedium.asOutParam())" at line 179 of file VBoxManageDisk.cpp
VBoxManage: error: Invalid UUID or filename "/Users/lawnmowerlatte/packer-debian/packer_cache/71496103e631b73270b67cf5011b37e64a93b062e305482536cc720ffea678c6.iso"

==> Builds finished but no artifacts were created.
```

**Comment by lawnmowerlatte** [comment]
Nevermind, the URL for `debian-jessie` is wrong. I'll create a separate issue for that.

**Comment by deimosfr** [comment]
Fixed in  commit 93d4ac9bb6c6f9578b6e2377ed88cf9f44410e09
